deploy/action.yml

```diff
@@ -127,7 +131,7 @@ runs:
     - name: Terraform output
       uses: actions/upload-artifact@v4
```

Comment:
I would also add the option to allow overwrite:
https://github.com/actions/upload-artifact#overwriting-an-artifact